deployer/templates/hawkular-cassandra-node-dynamic-pv.yaml

```diff
@@ -81,14 +81,9 @@ objects:
           - "--cluster_name=hawkular-metrics"
           - "--data_volume=/cassandra_data"
           - "--internode_encryption=all"
-          - "--require_node_auth=true"
+          - "--require_node_auth=false"
```

Comment:
That is the whole point of you looking into this. If we cannot use these certificates then there was no point in any of these commits.